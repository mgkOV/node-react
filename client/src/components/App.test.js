import React from 'react';
import { shallow } from 'enzyme';
import { Unwrapped as UnwrappedApp } from './App';

test('App should render correctly', () => {
  const component = shallow(<UnwrappedApp />);
  expect(component).toMatchSnapshot();
});
