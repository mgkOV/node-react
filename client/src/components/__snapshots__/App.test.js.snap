// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="container"
>
  <BrowserRouter>
    <div>
      <Connect(Header) />
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        exact={true}
        path="/surveys"
      />
      <Route
        component={[Function]}
        path="/surveys/new"
      />
    </div>
  </BrowserRouter>
</div>
`;
